admin/release-packaging/centos/development/clusterhq-release.spec

```diff
@@ -0,0 +1,44 @@
+# See README for instructions on how to build.
```

Comment:
I chose to do this because I thought that this will continue to have a README but the packaging file might get moved (so there is only one reference to change).